config.ini

```diff
@@ -0,0 +1,10 @@
+[DEFAULT]
+
```

Comment:
I think that we should use TOML, because it is better, according to https://www.python.org/dev/peps/pep-0518/#overview-of-file-formats-considered

Reply:
Done